### DISCUSSION TITLE
Several wasm apps on same server

### DISCUSSION THREAD

**Discussion by strnbrgd** [unoplatform/uno#16561] [Q&A]
Hello.
I've created Uno platform app, built as wasm it and run it in a docker container using nginx.
Now I want to run two Uno apps on the same nginx server. Is it possible?
I tried putting my apps in two different sub folders but then they won't work (each of them tries to get resources on the root folder....)
Can I place Uno wasm on sub folder of the server? is there another way to run two apps?

**Comment by jeromelaban** [comment]
Thanks for the question. It's possible yes, you need to set the base path using this: https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/features-deep-linking.html
